src/Assets/TestPackages/dotnet-dependency-context-test/dotnet-dependency-context-test.csproj

```diff
@@ -8,7 +8,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.DependencyModel" Version="$(MicrosoftExtensionsDependencyModelPackageVersion)" />
+    <!-- .NET 6 versions of this package no longer support .NET Core 2.2, so hard-code the version here to 5.0 -->
```

Comment:
I'm not sure what this means. The package supports `netstandard2.0`, why wouldn't that asset be used?

cc @ViktorHofer

Reply:
It looks like the latest build of DependencyModel now depends on System.Runtime.CompilerServices.Unsafe, and that package has a target in it blocking the build:

```xml
<Project InitialTargets="NETStandardCompatError_System_Runtime_CompilerServices_Unsafe_netcoreapp3_1">
  <Target Name="NETStandardCompatError_System_Runtime_CompilerServices_Unsafe_netcoreapp3_1"
          Condition="'$(SuppressTfmSupportBuildWarnings)' == ''">
    <Error Text="System.Runtime.CompilerServices.Unsafe doesn't support $(TargetFramework). Consider updating your TargetFramework to netcoreapp3.1 or later." />
  </Target>
</Project>
```

Reply:
OK, that explains it. Thanks.

Reply:
You're the first one that encounters these recent changes. I plan to send out a breaking change notice later this week after I synced up with @terrajobst on the format.

Is there a reason for dotnet/sdk to still have tests that build against an unsupported version of .NETCoreApp?

Reply:
@ViktorHofer Late reply: Generally we want the SDK to continue to build projects successfully even if they're out of support.  IE when a runtime goes out of support it means it won't get updates anymore, not that suddenly the tools stop working for that runtime.